### ISSUE TITLE
Subtitles no longer display with Random PSIdeas installed.

### ISSUE THREAD

**Issue by Birgliss** [Kamefrede/rpsideas#85]
### Versions:
Forge 14.23.5.2854
AutoRegLib 1.3-32
Forgelin 1.8.4
librarianlib 1.12.2-4.20-release
PSI r-1.1-77
Random PSIdeas 2.4.2
### Repro:
1) Install above mods
2) Go into sound settings, enable subtitles
3) Join a world (or create a new one)
4) Look sadly upon the blank corner where subtitles are supposed to go
![image](https://user-images.githubusercontent.com/20749209/79889222-b6669f80-83cb-11ea-8d76-f031074a3b85.png)

To confirm that this is a problem with Random PSIdeas and not any one of its many dependencies, I also tried to reproduce without the mod in the pack and just the library mods. Subtitles were in-fact displaying properly, which suggests either this mod or how it interacts with one of the dependencies.

I hope this can get fixed, because I'm not able to properly play Minecraft without subtitles enabled. They're an important accessibility tool.

**Comment by DD12-1916** [comment]
It appears that this an issue that started with version 2.3.0 of RandomPSIdeas. As such, downgrading to version 2.2.2 should fix it, which would of course mean that you would miss out newer content (and there are quite many), as well as several other important bug fixes.
I hope the developer gets on this as soon as they can, since subtitles are indeed important accessibility tools.

**Comment by DD12-1916** [comment]
So I found a fix; well, made a fix. I looked through the commits history and managed to identify the code change that caused this issue, which was the fix the developers made to address the following issues; #41, #42, #44, #47, and #48 on the 13th of March last year.
PotionFlashbang.java (from src/main/java/com/kamefrede/rpsideas/effect) was the one responsible for this issue. Since I have almost zero knowledge of coding whatsoever, I just simply copied the PotionFlashbang.java from version 2.2.2 (when the change was not yet made) over to version 2.4.2. Surprisingly, the game managed to launch without any crashes whatsoever and the subtitles were back.
This is obviously not recommended as it is not a proper fix and thus, bugs may very well prance to your house and cause mayhem, but if subtitles are really needed, this can be used until the developer comes around, which I hope won't be long.
However, it is just renders some overlay so I highly doubt it can wreck the entire universe and I think it would be a quick fix even for someone who has some knowledge in how this stuff works.

The .jar file is inside the .zip file so...

[rpsideas-2.4.2_unofficial.zip](https://github.com/Kamefrede/rpsideas/files/5080502/rpsideas-2.4.2_unofficial.zip)
